README.md

```diff
@@ -7,42 +7,69 @@ Sudachi & SudachiPy are developed in [WAP Tokushima Laboratory of AI and NLP](ht
 
 **Warning: SudachiPy is still under development, and some of the functions are still not complete. Please use it at your own risk.**
 
+## Breaking changes
+### v0.3.0
 
-## Setup
+- `resources/` directory was moved to `sudachipy/`.
+
+### V0.2.2
+
+- Distribute SudachiPy package via PyPI
+  - `pip install SudachiPy`
+
+### v0.2.0
+
+- User dictionary feature added 
+
+
+## Easy Setup
```

Comment:
For newcomers, `pip install SudachiPy` should be placed in Easy Setup.

Reply:
You mentioned users can install SudachiPy via pip... as text. However, if I was ML, NLP, Python beginner, just one line `pip install SudachiPy` changes impression.

Reply:
Thanks for suggestions.
I added SudachiDict_core pacakge URL to requirement part in setup.py.
https://github.com/megagonlabs/SudachiPy/pull/2

Now we can install SudachiPy and SudachiDict_core at the same time.
I changed Easy Setup description to execute `pip install SudachiPy`.

How do you think about this approach? > @izziiyt

Reply:
In conclusion, I can accept your idea. However I think we should create new issues for the bellow
___________________________________________
exact URL like this is one of the concerning points.
```
"SudachiDict_core @ https://object-storage.tyo2.conoha.io/v1/nc_2520839e1f9641b08211a5c85243124a/sudachi/SudachiDict_core-20190531.tar.gz",
```
For easy operation, in the future we need introduce approach like bellows
1. prepare proxy server to route exact URL
2. parse URL index file on web like nltk (https://raw.githubusercontent.com/nltk/nltk_data/gh-pages/index.xml)

We can say same thing to `README.md`. 
In my opinion, it can be solved easily so I can accept current your strategy.  
If we discuss with takaoka-san and other NLP staffs and it was impossible to apply easy URL control approach, we will change it at that time.  
_____________________________________________

Reply:
Thanks for your deep analysis and describing our future works.
I'd like to merge https://github.com/megagonlabs/SudachiPy/pull/2 to this PR.